### PR TITLE
added post on rails starter template

### PR DESCRIPTION
**Pull request by anujmiddha**

### CODE REVIEW

File: source/_posts/Rails-Starter-Template.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+---
+title: Rails Starter Template
```
Comment (Rajeev111974) [inline source/_posts/Rails-Starter-Template.md:2]: use few keyword from this list as we need for increasing the ranking of the page

NAYAN
Traffic monitoring
AI
Livelihood
Driver analytics
Violation mapping
Better commute
Monitor infrastructure defects
Safer roads
Road Safety
Traffic Violation
Artificial Intelligence
India
Smart City
Safety
Data Protection
PARKING DETECTION, 
VEHICLE OCCUPANCY DETECTION
NAYANINDIA
USER INTERACTION
NAYAN IP
NAYAN DATA PROTECTION
Road User
Public Road
Vehicle
Dangers
MACHINE LEARNING

File: source/_posts/Rails-Starter-Template.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+date: 2020-10-05 19:18:31
+author: Anuj Middha
+category: Rails
+tags:
+- Ruby
```
Comment (Rajeev111974) [inline source/_posts/Rails-Starter-Template.md:7]: Use 1-2 long tail keyword

File: source/_posts/Rails-Starter-Template.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
+---
```
Comment (Rajeev111974) [inline source/_posts/Rails-Starter-Template.md:1]: Use keywords throughout your article

Comment (Rajeev111974) [inline source/_posts/Rails-Starter-Template.md:1]: Link to previous Content - in content there should be link with previous content

Comment (Rajeev111974) [inline source/_posts/Rails-Starter-Template.md:1]: Keyword should be linked with website

Comment (Rajeev111974) [inline source/_posts/Rails-Starter-Template.md:1]: Content should have link to previous blog

**Comment by Rajeev111974** [comment]
HI Anuj JI,

 

Blog is reviewed and find some changes is there which we have mentioned online, Please check.

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

File: source/_posts/Rails-Starter-Template.md (hunk 1 above)
Comment (anujmiddha) [reply source/_posts/Rails-Starter-Template.md:2]: Keywords we can use only those that are relevant to the content of the blog. As none of these keywords are relevant here, I have added a line at the bottom of the blog mentioning "Nayan, Traffic monitoring and road safety"

File: source/_posts/Rails-Starter-Template.md (hunk 2 above)
Comment (anujmiddha) [reply source/_posts/Rails-Starter-Template.md:7]: Its not relevant, we cannot use non relevant keywords

File: source/_posts/Rails-Starter-Template.md (hunk 3 above)
Comment (anujmiddha) [reply source/_posts/Rails-Starter-Template.md:1]: Article should be based on content, not keywords

Comment (anujmiddha) [reply source/_posts/Rails-Starter-Template.md:1]: The content is not related to any previous content, hence not added link. Have added link to Nayan website.

**Pull request merged**